metals/src/main/scala/scala/meta/internal/metals/MetalsServerConfig.scala

```diff
@@ -144,9 +144,8 @@ object MetalsServerConfig {
       case "sublime" =>
         base.copy(
           isHttpEnabled = true,
-          // Sublime text opens an invasive alert dialogue for window/showMessage
-          // and window/showMessageRequest.
-          showMessage = ShowMessageConfig.logMessage,
+          showMessage = ShowMessageConfig.on,
```

Comment:
I would remove this line so that it uses the default as the other editors (the default is `"on"` unless overridden by a system property) 

Actually, we can probably remove `ShowMessageConfig` and `ShowMessageRequestConfig` altogether since as far as I know they were only used by Sublime Text.

I doubt anyone uses those system properties directly, and if they do, I would be curious to know why and better handle this on the server directly.

Reply:
(It's fine to just remove the line for this PR, I can follow up with a different PR cleaning up the config options)

Reply:
actually, I'll merge as is and follow up directly.